src/libstd/sync/mutex.rs

```diff
@@ -393,6 +393,19 @@ impl<T: ?Sized + fmt::Debug> fmt::Debug for Mutex<T> {
     }
 }
 
+#[unstable(feature = "std_display", issue = "24372")]
```

Comment:
All of these can be made `#[stable(feature = "std_guard_impls", since = "1.20")]` since stability on impls isn't tracked today. I think these will go into 1.20, but I could be wrong.